### ISSUE TITLE
Maui Blazor Hybrid doesn't Import JavaScript When UseMaui is Enabled in RCL

### ISSUE THREAD

**Issue by AathifMahir** [dotnet/maui#23149]
### Description

Importing JavaScript on Maui Blazor Hybrid Project would throw the below exception, This is Happens When the Razor Class Library Uses Maui.
![image](https://github.com/dotnet/maui/assets/17165402/9c4695e4-5512-45ca-bd92-69d78cb378fa)

### Link to public reproduction project repository

https://github.com/AathifMahir/MauiBlazorJsInteropTest

### Version with bug

8.0.40 SR5

### Is this a regression from previous behavior?

Not sure, did not test other versions

### Last version that worked well

Unknown/Other

### Affected platforms

Windows and Android, I was *not* able test on other platforms

### Affected platform versions

Windows 11 23H2

### Did you find any workaround?

Not able to find any workaround for now

**Comment by ninachen03** [comment]
I can repro this issue on Visual Studio 17.11.0 Preview 2.0 (8.0.60 & 8.0.3).

**Comment by AathifMahir** [comment]
I have Finally Found the Issue, Seems like url structure is different on MauiBlazorHybrid When Compared All other Blazor Flavors, I'm not sure this is an Intentional Change or Bug With MauiBlazorHybrid Causing it,

When it comes Importing a Javascript from wwwroot, We use go with Below Url on All the Blazor Flavors
```xml
./_content/{Project-Name}/{File-Name}.js
```
But on MauiBlazorHybrid, The Above Structure Doesn't Work Anymore, Instead We need to have like below
```xml
./{File-Name}.js
``` 

The Same Applies to Isolated javascript that Located Under Component, That also need to imported without using _content


**Question is Whether this is Intentional Behavior that Changed with Recent MauiBlazorHybrid Release or Some Bugs Causing this issue?**

**Comment by AathifMahir** [comment]
I Found the issue, This has do with MauiBlazorHybrid Copies All the Content Razor Class Libraries wwwroot to Main Project Root, Therefore using url syntext like below doesn't work anymore

```xml
"./_content/{project-name}/{file-name}.js"
```
instead url syntext without _content and project name does the job
```xml
"./{file-name}.js"
```

i'm not sure this an intended behavior or a bug, This Causes Inconsistencies Between Other Flavors of Blazor and MauiBlazorHybrid

**Comment by AathifMahir** [comment]
I Found the issue, This has do with Setting **UseMaui** to **True** on Razor Class Library, This makes all the content inside the wwwroot of Razor Class Library to be Copied to Root of Main Project instead of _content/{AssemblyName} folder, this makes url used for importing the javascript invalid since the files doesn't exist, In this case url is changed like below

**From**
```xml
"./_content/{assemblyName}/{filename}.js"
```
**To**
```xml
"./{filename}.js"
```

**i'm not sure this an intended behavior or bug in build pipeline that Causing the Issue, Moreover this causes Inconsistency of Url**

**Comment by AathifMahir** [comment]
@samhouts I don't think this issue is exclusive to Windows, This issue does happen on Andriod as well.

The Platforms that I Have Tested were:
**Windows and Android**

If you want, i can test it on MacCatalyst and IOS as well

**Comment by mkArtakMSFT** [comment]
@javiercn can you please look into this ? Thanks!

**Comment by javiercn** [comment]
This is the same issue as described here with the same solution:
Static Web Assets + Webview.Maui are not supported for class libraries. Webview.Maui is meant for the final application.

https://github.com/dotnet/maui/issues/25999#issuecomment-3693107689
